content/docs/examples/advanced-gateways/ingress-certmgr/index.md

```diff
@@ -176,3 +176,56 @@ Hello version: v1, instance: helloworld-5d498979b6-jp2mf
 {{< /text >}}
 
 Note that you have to use the `--insecure` flag as certificates issued by the "staging" ACME-endpoints aren't trusted.
+
+## Moving to production from staging
+
+Now to switch to the production `letsencrypt` issuer.  First we'll reapply the certificate.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: ingress-cert-staging
+  namespace: istio-system
+spec:
+  secretName: ingress-cert-staging
+  issuerRef:
+    name: letsencrypt
+    kind: ClusterIssuer
+  commonName: $INGRESS_DOMAIN
+  dnsNames:
+  - $INGRESS_DOMAIN
+  acme:
+    config:
+    - http01:
+        ingressClass: istio
+      domains:
+      - $INGRESS_DOMAIN
+---
+EOF
+{{< /text >}}
+
+
+```
```

Comment:
For all uses of triple-backticks, please switch to use our {{< text >}} construct. And for any command-lines, start the text block with a $

Thanks.

Reply:
and removed the triple backticks and replaced with the text block convention